hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -14,13 +15,20 @@ def new_install?
     !File.exist?(success_file)
   end
 
-  def ensure_package(package, state = 'installed')
-    unless ['installed', 'absent', 'latest'].include?(state)
-      fail_and_exit('Incorrect package state supplied to ensure_package')
-    end
+  def ensure_packages(packages, state = 'installed')
+    return if packages.empty?
 
+    code = "package { ['#{packages.join('\', \'')}']: ensure => #{state} }"
     bin_path = Kafo::PuppetCommand.search_puppet_path('puppet')
-    execute("#{bin_path} resource package #{package} ensure=#{state}")
+
+    logger.debug("Ensuring #{packages.join(', ')} to package state #{state}")
```

Comment:
I think this is nice to surface as info rather than debug since its a critical step in execution of the installer.

Reply:
IMHO the katello-installer is too verbose. The installer should be quiet unless you pass --verbose and this should show up then.

Reply:
Correct... but this to me is not debug information this is `info` level information that should show up in the installer as such. That is, if I set my log level to `info`, I should see this output in my opinion. 

Related is that all of our scenarios have their log level set to `debug` which feels wrong as the default (https://github.com/theforeman/foreman-installer/blob/develop/config/foreman.yaml#L17). That's a separate issue to discuss.

Reply:
> Correct... but this to me is not debug information this is `info` level information that should show up in the installer as such. That is, if I set my log level to `info`, I should see this output in my opinion.

So essentially:
```suggestion
    logger.info("Ensuring #{packages.join(', ')} to package state #{state}")
```

That makes sense to me.



> Related is that all of our scenarios have their log level set to `debug` which feels wrong as the default (https://github.com/theforeman/foreman-installer/blob/develop/config/foreman.yaml#L17). That's a separate issue to discuss.

I think I agree with that, but indeed another discussion.